### DISCUSSION TITLE
Support for ENV variable RABBITMQ_CLUSTER_PARTITION_HANDLING

### DISCUSSION THREAD

**Discussion by Jdd208** [docker-library/rabbitmq#680] [Q&A] [answered]
Used RABBITMQ_CLUSTER_PARTITION_HANDLING part of environment variables, but it didn't get apply. 'rabbitmqctl report' suggests default strategy 'ignore' is applied.
Why it's not working with the ENV variable?

Tested on Docker image:  rabbitmq:3.12.8-alpine

**Comment by lukebakken** [comment, marked as answer]
You should use the `rabbitmq.conf` configuration file and the `cluster_partition_handling` setting:

https://www.rabbitmq.com/partitions.html#automatic-handling
